### DISCUSSION TITLE
Visibilità dati personali condizionata ad approvazione Custodian

### DISCUSSION THREAD

**Discussion by Giosafatv** [globaleaks/globaleaks-whistleblowing-software#3729] [General]
Salve,
ho fatto un template dove se il segnalante sceglie di lasciare i dati personali si apre uno step che contiene i campi per tali dati (nome, cognome, etc.) ho utilizzato il tipo "Campo di testo singola linea" ma inviando la segnalazione il Recipient vede i dati in chiaro e non ha da fare la richiesta al Custodian, l' unico modo in cui sono riuscito a sottoporre tali dati ad approvazione del Custodian è usando una domanda da modello "Would you like to tell us who you are?" che però con la lingua italiana viene "Vuoi dirci chi se?" (manca la i), io preferirei comunque poter sottomettere il tipo "Campo di testo singola linea" all' approvazione del Custodian se fosse possibile. O in alternativa sapere se è possibile modificare il template "Vuoi dirci chi se?". Grazie

**Comment by Giosafatv** [comment]
Risolta con wa https://github.com/orgs/globaleaks/discussions/3646

**Reply by cvrboni** [reply]
posso chiederti come hai risolto? volevo anche io modificare la parte della identificazione dei dati, però non riesco a fare l'attivazione delle domande (se ad esempio metti si a farti identificare, non riesco a far comparire le caselle nome tipo di contatto ecc), bisogna aggiungere il js?

**Reply by Giosafatv** [reply]
ho creato un Passo apposito per inserire i dati tipo "dati personali", la prima cosa che ho messo nel Passo è una condizione di attivazione "Vuoi inserire i tuoi dati identificativi?: Si (Sufficiente)" quindi il Passo si attiva solo se il segnalante ha risposto Si alla domanda "Vuoi inserire i tuoi dati identificativi?" e a seguire la domanda "inserisci i tuoi dati personali" che avevo precedentemente creato nei Modelli di Domanda (vedi https://github.com/orgs/globaleaks/discussions/3646) Spero di esserti stato di aiuto
